rpcs3/Emu/RSX/rsx_decode.h

```diff
@@ -73,7 +63,7 @@ struct registers_decoder<NV4097_SET_VIEWPORT_HORIZONTAL>
 
 	static std::string dump(decoded_type &&decoded_values)
```

Comment:
Looks like some dump invocations can be moved to cpp file since you are changing them anyway.

Reply:
Also using && here is weird, `const decoded_type&` I would pass.

Reply:
Define "some".

Reply:
Full template specializations (`template<>`).

Reply:
In rsx_decode.h, not including specializains by macros and `template <>`
![image](https://user-images.githubusercontent.com/18193363/104808520-7f54db80-57ef-11eb-8c63-740b4953a298.png)
I'm afraid you have to be more specific, I won't overwrite so many specializaions.

Reply:
I meant at least those you changed.

Reply:
That's nearly half of them (you can search fmt::format count), it's way too much unrewarding work. I'm sure this file will be refactored several times in the future because it's very inefficient about how it handles it all. (for example decoded_type structure is completely redundent, was there from legacy reasons where it was some sort of union).
I have to disagree doing this in this pull request.

Reply:
IIRC MSVC had a feature to do this automatically.